pkg/db/db_local/install.go

```diff
@@ -434,6 +434,16 @@ func initDatabase() error {
 	utils.LogTime("db_local.install.initDatabase start")
 	defer utils.LogTime("db_local.install.initDatabase end")
 
+	// initdb sometimes fail due to invalid locale settings, to avoid this we update
+	// the locale settings to use 'C' only for the initdb process to complete, and
+	// then return back to the existing locale settings of the user.
```

Comment:
you need to read the previous value of LC_ALL and reset it - only unset if there is no original value